EssayMonkey.py

```diff
@@ -0,0 +1,101 @@
+'''
+Daniel Edwards
+10/27/2015
+EssayMonkey.py
+A program that generates an essay based on input number of paragraphs
+and sentences per paragraph.
+
+This program assumes text files are in the folder matching the expected
+naming convention.
+'''
+import random #imported for randomizing sentence length
+
+'''
+This function takes number of paragraphs and sentences per paragraph as
+input and generates an essay matching those characteristics based on
+text files containing a list of nouns, verbs and adjectives.
+@param aParagraphs number of paragraphs
+@param aSentences number of sentences per paragraph
+@return Returns the string containing the completed essay.
+'''
+def essayMonkey(aParagraphs, aSentences):
+    tPCount = 0
+    tStr = ""
+    #generate lists
+    tWordList = createWordLists()
+    tNouns = tWordList[0]
+    tVerbs = tWordList[1]
+    tAdjec = tWordList[2]
+
```

Comment:
Instead of reading all 3 files into an array then splitting them back out it would make more sense to change createWordList function to take a file path and return an array. Then call the function 3 times.